resource_management_mobile/src/app/pages/main/main.component.scss

```diff
@@ -0,0 +1,7 @@
+.topbar,.footer{
```

Comment:
Can you move this topbar and .footer code to corresponding scss file - seems you have created a page for this

Reply:
Added scss on the respected files.